### PR TITLE
Pls check

### PR DESCRIPTION
**Pull request by Codebox124**
\* **Checkist**

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas

- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Reference to issue (if any) 

* **what should reviewer look at ?

* **where is the main change at ?

* **Other information**:

reviwer: @jaekralj @queen-techie

### CODE REVIEW

File: src/components/about us/AboutUs.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { Link } from 'react-router-dom'
+import fruit from '../../assets/img/fruit2.png'
```
Comment (queen-techie) [inline src/components/about us/AboutUs.jsx:2]: add these images to the index.jsx file in the asset folder and import it from there

**Comment by queen-techie** [comment]
the footer  file is empty kindly remove it at every instance it was used

File: src/components/about us/WhyChooseUs.jsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
+import icon from '../../assets/img/icon.png'
```
Comment (queen-techie) [inline src/components/about us/WhyChooseUs.jsx:1]: same here

**Review by queen-techie — CHANGES_REQUESTED** [review]

**Comment by queen-techie** [comment]
also give your PR a descriptive name

File: src/pages/AboutUs.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        <section>
+          <Hero />
+        </section>
+        <section>
+          <Aboutus />
+        </section>
+        <section>
+          <WhyChooseUs />
+        </section>
+        <section>
```
[the remark this answers is not in the document]
Comment (Codebox124) [reply src/pages/AboutUs.jsx:26]: done

File: src/components/about us/WhyChooseUs.jsx (hunk 2 above)
Comment (Codebox124) [reply src/components/about us/WhyChooseUs.jsx]: done

File: src/components/about us/WhyChooseUs.jsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+       </div>
+ 
+ 
+ 
+         </div>
+
+         <div className='grid grid-cols-4 gap-5 px-9'>
+            <div className='bg-white px-3 py-16  rounded-2xl text-center'>
+                <div className='flex justify-center'>
+                <img src={icon2} alt="" />
```
[the remark this answers is not in the document]
Comment (Codebox124) [reply src/components/about us/WhyChooseUs.jsx]: done

**Pull request closed without merging**